.vsts-ci.yml

```
@@ -32,7 +32,8 @@ variables:
     - group: DotNetBuilds storage account read tokens
     - name: _InternalRuntimeDownloadArgs
       value: /p:DotNetRuntimeSourceFeed=https://dotnetbuilds.blob.core.windows.net/internal 
-        /p:DotNetRuntimeSourceFeedKey=$(dotnetbuilds-internal-container-read-token-base64)
+        /p:DotNetRuntimeSourceFeedKey=$(dotnetbuilds-internal-container-read-token-base64) 
+        /p:OfficialBuild=true
```

```suggestion
        /p:OfficialBuilder=Microsoft
```
The suggestion in the comment was to call this `OfficialBuilder` and set it to `Microsoft`